### PR TITLE
Postgres with private endpoint

### PR DESCRIPTION
**Pull request by ArielRam99**
Creating Postgres DB with private endpoint. The DB is able to be connected to using the private IP address from the Private endpoint from a VM inside the virtual network.

### CODE REVIEW

File: steps/deployment/portgress-resource-deployment/postgres.tf [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+variable "location" {
+    default = "EastUS"
+}
+
+variable "administrator_login" {
+    default = "psqladmin"
+}
+
+variable "administrator_login_password" {
+    default = "H@Sh1CoR3!"
```
Comment (mosabami) [inline steps/deployment/portgress-resource-deployment/postgres.tf]: please change the password to 12345ASDf

File: steps/deployment/portgress-resource-deployment/postgres.tf [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+#########################################################################################
+
+variable "location" {
+    default = "EastUS"
+}
+
+variable "administrator_login" {
+    default = "psqladmin"
```
Comment (mosabami) [inline steps/deployment/portgress-resource-deployment/postgres.tf]: please make this postgres

**Review by mosabami — CHANGES_REQUESTED** [review]

File: steps/deployment/portgress-resource-deployment/postgres.tf (hunk 1 above)
Comment (ArielRam99) [reply steps/deployment/portgress-resource-deployment/postgres.tf]: @mosabami Updated

File: steps/deployment/portgress-resource-deployment/postgres.tf (hunk 2 above)
Comment (ArielRam99) [reply steps/deployment/portgress-resource-deployment/postgres.tf]: @mosabami Updated

**Review by mosabami — APPROVED** [review]

**Pull request merged**